meson.py

```diff
@@ -15,12 +15,10 @@
 # limitations under the License.
 
 from mesonbuild import mesonmain
-import sys, os
+import os
```

Comment:
__[Flake8]__

[F401] 'os' imported but unused


<!-- Comment by [SideCI](https://sideci.com) -->

Reply:
heh, I just nocide the same right before you posted :).